nengo_dl/__init__.py

```diff
@@ -20,7 +20,7 @@
 
 There are two options for getting NengoDL working:
 
-- Upgrade to Python >= 3.5
+- Upgrade to Python >= 3.6
 
 - Install an older version of NengoDL:
```

Comment:
I'll change this "older version of NengoDL" to 3.4.0, since that's the latest to support Python 3.5 (if they're using earlier than 3.5, they'll have to figure things out themselves).